jenkins/debian/debos/scripts/crush.sh

```diff
@@ -6,151 +6,181 @@ set -e
 # IMPORTANT: The Debian system is not longer functional at this point,
 # for example, apt and dpkg will stop working
 
-UNNEEDED_PACKAGES="apt libapt-pkg5.0 "\
-"ncurses-bin ncurses-base libncursesw5 libncurses5 "\
-"perl-base "\
-"debconf libdebconfclient0 "\
-"libfdisk1 "\
-"insserv "\
-"init-system-helpers "\
-"cpio "\
-"passwd "\
-"libsemanage1 libsemanage-common "\
-"libsepol1 "\
-"gzip "\
-"gnupg "\
-"gpgv "\
-"hostname "\
-"adduser "\
-"debian-archive-keyring "\
 
+filesystem()  {
+   UNNEEDED_PACKAGES="libfdisk1 cpio hostname gzip "
```

Comment:
`hostname` and `gzip` are not file system tools. `cpio` isn't exactly one either even though it's often used for initrd.

Reply:
moved `gzip` and `cpio` under `compress_tools()` and `hostname` to `hostname_tool()`